applications/migrations/0026_auto_20210225_0850.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 3.1 on 2021-02-25 06:50
```

Comment:
Would like to see the automatically generated migration filename to be renamed to something meaningful. For example `0026_add_rejected_application_status.py`.

Reply:
Good catch @klemmari1 👍  Let's do it like that @haavikko

Reply:
fixed